index.js

```diff
@@ -183,4 +183,30 @@ form.addEventListener('submit', (e) => {
     msg.style.visibility = 'visible';
     document.getElementById('email').classList.add('email_validation');
   }
-});
\ No newline at end of file
+});
+
+// preserve data in the browser
+
+const { localStorage } = window;
+
```

Comment:
- [ ] Your local storage work fine, however, you think of clearing the input after the form was submitted, the local storage help the user not to lose the data before he validate the action, but once it is done, I think you don't need to keep again the input data prefilled since the form was already submitted, please fix this on the go @AjroudiRida